docs/source/programming_resources/tutorial_specific/blocks/creating_op_modes/Writing-an-Op-Mode-with-FTC-Blocks.rst

```diff
@@ -1,5 +1,5 @@
-Creating Op Modes (Blocks)
-==============================
+Creating Op Modes :bdg-warning:`Blocks`
+========================================
```

Comment:
Let's make headings tags equal to the content above them. I know the specification says "at least as long" but we have thus far been really good about "exactly the length".